index.html

```diff
@@ -255,12 +255,6 @@ <h4 class="title title--small trn">Weekly schedule for affixes</h4>
                             	<td class="table__cell"><span class="trn">Tyrannical</span></td>
                                 <td class="table__cell"><span class="trn">Raging</span></td>
                                 <td class="table__cell"><span class="trn">Necrotic</span></td>
-                                <td class="table__cell"><span class="trn">Awakened</span></td>
-							</tr>
```

Comment:
These are from the previous line in the table, so they should stay.

Reply:
This is just the UI telling you that its taken away but the net result is exactly the same.